directory.packages.props

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageVersion Include="Nuke.Common" Version="6.0.1" />
+    <PackageVersion Include="NuGet.CommandLine" Version="5.8.1" />
```

Comment:
Let's remove unused dependencies not needed yet. Eg the Nuke/Resharper stuff

Reply:
removed